qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst

```diff
@@ -43,23 +36,95 @@ Here we generate a single BIOM table with the OTUs/per-sample. The OTU IDs are g
 
 Currently, we have the reference databases: Greengenes version 3_8-97, Silva 119 and Unite 7. Depending on your selection is if the reference has a phylogenetic tree.
 
-
 Shotgun sequencing
 ------------------
 
-Qiita currently has one shotgun metagenomics data analysis pipeline. Note that this is the initial processing pipeline and we will be adding more soon.
-
-With that said, the current workflow is as follows:
-
-#. Removal of adapter sequence and host contamination using `KneadData <https://bitbucket.org/biobakery/kneaddata/wiki/Home>`_.
-#. Gene calling and pathway profiling using `HUMAnN2 <https://bitbucket.org/biobakery/humann2/wiki/Home>`_.
-
-This workflow starts with per_sample_FASTQ files. We recommend only uploading sequences that have already been through QC and host /
-human sequence removal. However, all sequence files currently are required to go through KneadData to ensure they are ready for
-subsequent analyses. Currently, the KneadData command removes adaptor sequences (choice of TruSeq3-PE-2 and NexteraPE-PE) and
-sequences mapping to the human genome (additional host genomes will become available soon).
-
-Next, the QC'd sequences will be compared against reference databases to determine the presence and abundance of protein-coding functional genes and
-pathways using HUMAnN2. These are then summarized as BIOM tables, which can be used in subsequent analysis and visualization.
-
-For more information visit the `Shotgun Qiita Plugin GitHub page <https://github.com/qiita-spots/qp-shotgun>`_.
+Qiita currently has one shotgun metagenomics data analysis pipeline: `Shogun <https://msystems.asm.org/content/3/6/e00069-18>`_.
+
+The current workflow is as follows:
+
+#. Removal of adapter sequence and quality control: `Atropos <https://github.com/jdidion/atropos/>`_
+#. Host contamination using `Bowtie2 <http://bowtie-bio.sourceforge.net/bowtie2/index.shtml>`_
+#. Taxonomy profiling using three different aligners and two different reference databases; see sections below
```

Comment:
How about "...using **choice of** three different..."

Reply:
thanks